Hestia/app/src/main/res/values/strings.xml

```diff
@@ -49,4 +49,16 @@
     <string name="passSet">Password set to: </string>
     <string name="userSet">Username set to: </string>
     <string name="userNotSet">Username not changed (length under 5)</string>
+    <string name="deviceEndpoint">devices/</string>
+    <string name="activatorEndpoint">/activators/</string>
+    <string name="pluginEndpoint">plugins/</string>
+    <string name="loginEndpoint">login/</string>
+    <string name="ioExceptionMessage">Could not connect to the server</string>
+    <string name="enterIpHint">Enter ip here</string>
+    <string name="credentialsCorrectlyChanged">Credentials successfully changed</string>
+    <string name="addDeviceTitle">new Add device</string>
```

Comment:
As stated earlier, "new Add device" is quite a strange name for a dialog. Simply "Add device" should do in my opinion